client/me/help/main.jsx

```diff
@@ -108,9 +144,13 @@ const Help = React.createClass( {
 } );
 
 module.exports = reactRedux.connect(
-	( state ) => {
+	( state, ownProps ) => {
+		const isEmailVerified = currentUser.isCurrentUserEmailVerified( state );
+		const showCoursesTeaser = ownProps.isCoursesEnabled;
+
 		return {
-			isEmailVerified: currentUser.isCurrentUserEmailVerified( state ),
+			showCoursesTeaser,
+			isEmailVerified,
 		};
 	}
 )( Help );
```

Comment:
> Although I guess showCoursesTeaser might not necessarily === ownProps.isCoursesEnabled in the future?

Thats correct, It'll be composed of another boolean that indicates if the user has the business package.